rest-api/test/client_test/participant_test.py

```diff
@@ -57,7 +62,8 @@ def test_create_and_modify_participant(self):
           headers = { 'If-Match': last_etag})
     self.assertEqual(response['biobankId'], biobank_id)
 
-    self.assertJsonEquals(response['providerLink'], [ provider_link_2])
+    self.assertJsonEquals(response['providerLink'], [provider_link_2])
```

Comment:
It's a little confusing to re-post response above like this... might be clearer to modify providerLink on new_response and send that in the PUT?

Reply:
changed.